src/app/records/tiled-records.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, OnInit, Injector, ChangeDetectionStrategy } from '@angular/core';
+import { RecordDataForSearch, RecordResult, RecordQueryFilters, CustomFieldDetailed, RecordLayoutEnum } from 'app/api/models';
+import { RecordsService } from 'app/api/services';
+import { HttpResponse } from '@angular/common/http';
+import { Observable, BehaviorSubject } from 'rxjs';
+import { BasePageComponent } from 'app/shared/base-page.component';
+import { HeadingAction } from 'app/shared/action';
+import { RecordHelperService } from 'app/core/records-helper.service';
+
+type RecordSearchParams = RecordQueryFilters & { owner: string, type: string };
+
+@Component({
+  selector: 'tiled-records',
+  templateUrl: 'tiled-records.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+
+export class TiledRecordsComponent
+  extends BasePageComponent<RecordDataForSearch>
+  implements OnInit {
+
+  type: string;
+  param: string;
+  fieldsInList: Array<CustomFieldDetailed>;
+  results$ = new BehaviorSubject<RecordResult[]>(null);
+
+  constructor(
+    injector: Injector,
+    private recordsService: RecordsService,
+    private recordsHelper: RecordHelperService
+  ) {
+    super(injector);
+  }
+
+  ngOnInit() {
+    super.ngOnInit();
+
+    this.param = this.route.snapshot.params.owner;
+    this.type = this.route.snapshot.params.type;
+
+    this.addSub(this.recordsService.getRecordDataForOwnerSearch({ owner: this.param, type: this.type }).subscribe(data => {
+      if (data.type.layout !== RecordLayoutEnum.TILED) {
+        throw new Error(`Invalid record layout: ${data.type.layout}`);
+      }
+
+      // TODO handle profileFields
```

Comment:
It will never have profile fields filters in tiled records, because they are only used in general search.
Even tiled records are shown as list in general search.
Tiled layout is only used in records for a specific owner and type.